.circleci/config.yml

```diff
@@ -126,7 +129,15 @@ references:
             helm dep up ./codacy
             echo "Adding 'https://charts.codacy.com/${CHANNEL}'"
             helm repo add --username "${CHARTS_REPO_USER}" --password "${CHARTS_REPO_PASS}" codacy https://charts.codacy.com/${CHANNEL}
-            helm push ./codacy codacy
+
+            if "$CHANNEL" == "stable"; then
```

Comment:
this looks very hacky. This entire anchor just stopped being generic this way.
maybe just change the `.version` file in the workflow before calling this step?

Reply:
I agree. I have extracted this into a separate job that we call in the workflow.